<?xml version="1.0"?>

<launch>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="framerate" value="10" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>

  <node name="text_detection" pkg="text_detection" type="text_detection" output="screen">
    <param name="tessdata_path" value="/home/strymj/tesseract/tessdata/" />
    <param name="language" value="eng" />
    <param name="camera_framerate" value="10" />
  </node>

</launch>
